pages/palette/[id].tsx

```diff
@@ -43,8 +43,11 @@ const Instructions = styled.legend`
 
 const LayoutSwitch = styled.div`
   align-items: center;
+  background-color: ${(props) => props.theme.colours.white};
+  border-radius: 0.5rem;
   display: flex;
   flex-direction: column;
+  padding: 1rem;
```

Comment:
This affects the alignment on smaller screens so there aren't the same number of colours in a line.

Reply:
I'll add this to #46